### PR TITLE
Documentation consistency

### PR DESCRIPTION
**Pull request by rpspringuel**
This PR corresponds with gregorio-project/gregorio#1390 and is to keep the website consistent with the updated gabc documentation there.

### CODE REVIEW

File: gabc/index.html [hunk 1]
```diff
@@ -146,7 +148,69 @@ <h3 id="header">Header</h3>
...
 </dt>
 
+<dd><a name="author">author</a></dd>
+<dt>The author of the piece, if known; of course, the author of most
+traditional chant is not known.</dt>
+
+<dd><a name="language">language</a></dd>
+<dt>The language of the lyrics.</dt>
+
+<dd><a name="language">oriscus-orientation</a></dd>
```
Comment (henryso) [inline gabc/index.html]: This anchor name should probably not be language.

File: gabc/index.html [hunk 2]
```diff
@@ -146,7 +148,69 @@ <h3 id="header">Header</h3>
...
+<dt>If <code>legacy</code>, the orientation of an unconnected oriscus must be set manually.</dt>
+
+<dd><a name="mode">mode</a></dd>
+<dt>The mode of the piece. This should normally be an arabic
```
Comment (henryso) [inline gabc/index.html]: I think "arabic" should be capitalized.

File: gabc/index.html [hunk 3]
```diff
@@ -146,7 +148,69 @@ <h3 id="header">Header</h3>
...
+	number between 1 and 8, but may be any text required for unusual
+	cases. The mode number will be converted to roman numerals and
```
Comment (henryso) [inline gabc/index.html]: "roman" should probably be capitalized as well.

File: gabc/index.html [hunk 4]
```diff
@@ -146,7 +148,69 @@ <h3 id="header">Header</h3>
...
+	overwrite that definition.
+</dt>
 
+<dd><a name="staff-lines">staff-lines</a></dd>
+<dt>The number of lines on the staff.  You can have as few as 2 lines and as many as 5.  If this header is left out, then the default will be the classic 4 line staff.
+</dt>
+
+</dl>
+
+<p>Although gregorio ascribes no special meaning to them, other suggested headers are:</p>
```
Comment (henryso) [inline gabc/index.html]: I ran into this when doing the TeX documentation.  I think "gregorio" should be capitalized here as it refers to the system on a whole as opposed to the "gregorio" executable.  I've been trying to be consistent in "Gregorio" meaning the system as a whole, "GregorioTeX" being the tex portion, and "gregorio" being the executable.  What do you think?

**Review by henryso — CHANGES_REQUESTED** [review]
Just minor stuff.  It looks generally fine.

File: gabc/index.html (hunk 4 above)
Comment (rpspringuel) [reply gabc/index.html]: I think that's a reasonable standard and will fix this.

**Review by henryso — APPROVED** [review]

**Pull request merged**